scalecube-organization-api/src/main/java/io/scalecube/account/api/GetMembershipRequest.java

```diff
@@ -4,13 +4,16 @@
 
   private Token token;
 
-  public GetMembershipRequest() {}
```

Comment:
I think we need to revert it. But you can change its modifier from public to package-private

Reply:
done